samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,95 @@
+#. It boots the application on the network core.
+     After possibly performing a firmware update, and enabling flash memory protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
 
 Requirements
 ************
 
-  * |nRF5340DK|
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpunet
 
 .. _net_bootloader_build_and_run:
 
 Building and running
 ********************
 
-The source code of the sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
+The recommended way of building the sample is to include it as a child image in a :ref:`multi-image <ug_multi_image>` build where there is an application that runs on the network core, rather than building it stand-alone.
+There are two ways to include it in a multi-image build:
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+* If you don't need the capability to update the sample, enable the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the network core.
+* If you need the capability to update the sample, enable the :option:`CONFIG_BOOTLOADER_MCUBOOT` option in the application that runs on the application core to add MCUboot to the build.
+  When doing so, the build system includes the sample in the build by enabling automatically the :option:`CONFIG_SECURE_BOOT` option for the application that runs on the network core.
+  In addition, you must also enable the :ref:`subsys_pcd` library for MCUboot by setting the :option:`CONFIG_PCD` option when building its image.
+  See :ref:`ug_multi_image` for more information about setting configurations to a specific image.
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+  .. note::
+     In this setup, MCUboot verifies any new image received through a device firmware update (DFU) transport layer, like a serial or a Bluetooth LE connection, for both the application and the network core.
+     This includes the network core bootloader image.
```

Comment:
No, the network core bootloader image is never verified. It is immutable and does not need to be verified.

Reply:
fixed